src/Components/Components/src/ComponentResolver.cs

```diff
@@ -14,22 +14,22 @@ namespace Microsoft.AspNetCore.Components
     internal static class ComponentResolver
     {
         /// <summary>
-        /// Lists all the types 
+        /// Lists all the types
         /// </summary>
         /// <param name="appAssembly"></param>
         /// <returns></returns>
         public static IEnumerable<Type> ResolveComponents(Assembly appAssembly)
         {
-            var blazorAssembly = typeof(IComponent).Assembly;
+            var componentAssembly = typeof(IComponent).Assembly;
```

Comment:
Nit: `componentsAssembly` would make more sense, since this specifically references `Microsoft.AspNetCore.Components`.

Reply:
fixed